.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - run: npm install -g truffle
```

Comment:
[solidity-coverage](https://github.com/sc-forks/solidity-coverage) doesn't support latest ganache version 
problem: https://github.com/sc-forks/solidity-coverage/issues/694
***
I see two options:
1) Use an older version of truffle `@5.5.22` 
2) Use the Hardhat framework.